medocker/docker_shared/ubuntu_packages.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+set -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update -y
+
+apt-get install -y \
+    automake            \
+    build-essential     \
+    ca-certificates     \
+    ccache              \
+    clang               \
+    cmake               \
+    gcc                 \
+    gfortran            \
+    git                 \
+    gnupg               \
+    llvm-9-dev          \
+    libblosc-dev        \
+    libffi-dev          \
+    libfreetype6-dev    \
+    libhdf5-dev         \
+    liblapack-dev       \
+    liblapacke-dev      \
```

Comment:
Just want to make sure that `liblapack-dev` and `liblapacke-dev` are separate packages and not a typo or something.

Reply:
Looks like, see https://packages.debian.org/sid/liblapacke-dev